### PR TITLE
Fix - Moved Azure secret key-value pairs to base configs

### PR DESCRIPTION
**Pull request by channa-patil**
### Jira link

See [PROJ-XXXXXX](https://tools.hmcts.net/jira/browse/PROJ-XXXXXX)

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

## 🤖AEP PR SUMMARY🤖

_I'm a bot that generates AI summaries of pull requests, see [AEP](https://kainossoftwareltd.github.io/ai-enhanced-platform/) for more details_

- aat.yaml
  - Removed the schedule field from the job environment.

- demo-image-policy.yaml
  - Removed filterTags and policy from the spec section of the file.

- demo.yaml
  - Updated the image field with a new image reference.
  - Removed the keyVaults and secrets from the job section.

- prl-cron-fm5-reminder.yaml
  - Updated the schedule from \"25 17 * * *\" to \"0 7 * * *\".

- prl-cron-hwf-process-awp/demo.yaml 
  - Removed the keyVaults and secrets from the job section.

- prl-cron-hwf-process-awp/prl-cron-hwf-process-awp.yaml
  - Removed the keyVaults and secrets from the job section.

### CODE REVIEW

**Review by AlokDatta — APPROVED** [review]

File: apps/private-law/prl-cron-fm5-reminder/demo.yaml [hunk 1]
```diff
@@ -7,54 +7,7 @@ spec:
...
-            - name: prl-rd-api-system-username
-              alias: PRL_IDAM_REFDATA_USERNAME
-            - name: prl-rd-api-system-user-password
-              alias: PRL_IDAM_REFDATA_PASSWORD
-            - name: welsh-court-mapping
-              alias: WELSH_COURT_MAPPING
-            - name: app-insights-connection-string
-              alias: app-insights-connection-string
-      image: hmctspublic.azurecr.io/prl/cos:pr-2544-e809114-20240912102254 #{"$imagepolicy": "flux-system:demo-prl-cron-fm5-reminder"}
+      image: hmctspublic.azurecr.io/prl/cos:prod-835c14b-20240911080538 #{"$imagepolicy": "flux-system:prl-cos"}
```
Comment (AlokDatta) [inline apps/private-law/prl-cron-fm5-reminder/demo.yaml:10]: prl-cos is it valid?

**Review by AlokDatta — CHANGES_REQUESTED** [review]

File: apps/private-law/prl-cron-fm5-reminder/demo-image-policy.yaml [hunk 2]
```diff
@@ -2,14 +2,6 @@ apiVersion: image.toolkit.fluxcd.io/v1beta1
 kind: ImagePolicy
 metadata:
   name: demo-prl-cron-fm5-reminder
-  annotations:
-    hmcts.github.com/prod-automated: disabled
```
Comment (AlokDatta) [inline apps/private-law/prl-cron-fm5-reminder/demo-image-policy.yaml:6]: If you are deploying image prl-cos, you dont need to make changes to this file

File: apps/private-law/prl-cron-fm5-reminder/demo.yaml (hunk 1 above)
Comment (channa-patil) [reply apps/private-law/prl-cron-fm5-reminder/demo.yaml:10]: Yes @AlokDatta, it will refer prl-cos & pull latest prod image

**Review by AlokDatta — CHANGES_REQUESTED** [review]

File: apps/private-law/prl-cron-fm5-reminder/demo-image-policy.yaml (hunk 2 above)
Comment (channa-patil) [reply apps/private-law/prl-cron-fm5-reminder/demo-image-policy.yaml:6]: Yes, we don't need these config as FM5 changes are merged to master already.

Comment (channa-patil) [reply apps/private-law/prl-cron-fm5-reminder/demo-image-policy.yaml:6]: This is just to revert config added to deploy PR image

**Review by AlokDatta — APPROVED** [review]

**Pull request merged**